lxd/daemon.go

```diff
@@ -1017,6 +1018,9 @@ func (d *Daemon) Ready() error {
 	// Restore containers
 	containersRestart(s)
 
+	// Start monitoring VMs again
```

Comment:
@stgraber the "again" bit here caught my eye, what does it mean?

Reply:
It stopped when the LXD daemon shut down so has to be restarted for all running VMs on startup.